### PR TITLE
Bump Zeebe to 8.6.0

### PR DESCRIPTION
**Pull request by tmetzke**
## Description
* Bumps the version in the Java guide
* Does not adjust any other guide
  * Go client has moved to the community - needs clarification from @megglos what to do with this in the future.
  * Spring has a [newer dedicated guide](https://github.com/camunda/camunda-8-get-started-spring) - will be clarified in https://github.com/camunda/developer-experience/issues/406

### CODE REVIEW

**Review by berkaycanbc — COMMENTED** [review]
@tmetzke does changes in Spring blocking us from bumping its version to 8.6.0? I think, we can just move forward with bumping Spring version and make changes later when a decision made there.

For Go client, again, we can tackle (possibly remove) it in a separate issue.

**Comment by tmetzke** [comment]
> @tmetzke does changes in Spring blocking us from bumping its version to 8.6.0? I think, we can just move forward with bumping Spring version and make changes later when a decision made there.
> 
> For Go client, again, we can tackle (possibly remove) it in a separate issue.

I would postpone to adjust the Spring guide altogether now, to be honest. It uses the community SDK still, not the Camunda SDK. I would treat this the same way we treat the Go, C#, JS, etc guides: this is not our responsibility because we don't own this.

**Comment by tmetzke** [comment]
Bumping to 8.6.1 already.

File: java/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ provides a Zeebe client.
 <dependency>
 	<groupId>io.camunda</groupId>
 	<artifactId>zeebe-client-java</artifactId>
-	<version>8.5.6</version>
+	<version>8.6.0</version>
```
Comment (tmetzke) [inline java/README.md:16]:
```suggestion
	<version>8.6.1</version>
```

File: java/pom.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
   <properties>
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
-    <zeebe.version>8.5.6</zeebe.version>
+    <zeebe.version>8.6.0</zeebe.version>
```
Comment (tmetzke) [inline java/pom.xml:14]:
```suggestion
    <zeebe.version>8.6.1</zeebe.version>
```

**Comment by megglos** [comment]
Superseded by holistic update of all samples to 8.6.x #349 

Thanks for triggering the alignment on future maintenance.

**Pull request closed without merging**